docs/CLI.md

```diff
@@ -380,6 +394,20 @@ jest --shard=3/3
 
 Print your Jest config and then exits.
 
+### `--showSeed`
+
+Prints the seed value in the test report summary. See [`--seed=<num>`](#--seednum) for the details.
+
+An example output, the first line shows the seed used.
+```
+Seed:        1831310091
+Test Suites: 2 failed, 5 passed, 7 total
+Tests:       15 failed, 11 skipped, 53 passed, 79 total
+Snapshots:   0 total
+Time:        1.138 s
+Ran all test suites.
+```
```

Comment:
Just an opinion – for me this example feels redundant. The printout might change with time, e.g. something might be added. It will be hard to remember to update the text here.

Of course, it is nice to illustrated documentation with examples to make it more attractive for the reader. But this one feels unnecessary for my eye.

```suggestion
```